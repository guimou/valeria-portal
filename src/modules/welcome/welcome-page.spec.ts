import { createLocalVue, shallowMount } from '@vue/test-utils';
import Vue, { VueConstructor } from 'vue';
import WelcomePage from './welcome-page';

describe('WelcomePage', () => {
    let localVue: VueConstructor<Vue>;

    beforeEach(() => {
        localVue = createLocalVue();
    });

    it('should render correctly', () => {
        expect(true).toBeTruthy();

        const rdo = shallowMount(WelcomePage, { localVue: localVue });

        return expect(rdo.html()).toMatchSnapshot();
    });

});
