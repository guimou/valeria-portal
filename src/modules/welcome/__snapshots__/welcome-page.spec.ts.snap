// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomePage should render correctly 1`] = `
<section>
  <section class="welcome-page-headline">
    <article class="welcome-page-headline__body">
      <div>
        <h1 class="welcome-page-headline__title">welcome:modul</h1>
        <p class="welcome-page-headline__subtitle">welcome:tagline</p>
      </div>
      <table class="welcome-page-logo">
        <tr>
          <td class="welcome-page-logo__item"></td>
          <td class="welcome-page-logo__item wp--is-yellow"></td>
          <td class="welcome-page-logo__item wp--is-red"><span></span></td>
        </tr>
        <tr>
          <td class="welcome-page-logo__item"></td>
          <td class="welcome-page-logo__item wp--is-yellow"></td>
          <td class="welcome-page-logo__item wp--is-yellow"></td>
        </tr>
        <tr>
          <td class="welcome-page-logo__item"></td>
          <td class="welcome-page-logo__item"></td>
          <td class="welcome-page-logo__item wp--is-yellow"></td>
        </tr>
      </table>
    </article>
  </section>
  <section class="welcome-page__content">
    <m-button-stub type="button" skin="primary" iconposition="left" iconsize="12px">
      <m-icon-stub name="edit" size="1em"></m-icon-stub> Modul button
    </m-button-stub>
    <p>
      <m-accordion-group-stub skin="default" class="m-u--margin-top">
        <m-accordion-stub skin="default" iconposition="left" iconsize="large" padding="true" paddingheader="true" paddingbody="true">
          <h2 class="m-u--no-margin">Titre 1</h2>
          Contenu de l'accordéon 1
        </m-accordion-stub>
        <m-accordion-stub skin="default" iconposition="left" iconsize="large" padding="true" paddingheader="true" paddingbody="true">
          <h2 class="m-u--no-margin">Titre 2</h2>
          Contenu de l'accordéon 2
        </m-accordion-stub>
        <m-accordion-stub skin="default" iconposition="left" iconsize="large" padding="true" paddingheader="true" paddingbody="true">
          <h2 class="m-u--no-margin">Titre 3</h2>
          Contenu de l'accordéon 3
        </m-accordion-stub>
        <m-accordion-stub skin="default" iconposition="left" iconsize="large" padding="true" paddingheader="true" paddingbody="true">
          <h2 class="m-u--no-margin">Titre 4</h2>
          Contenu de l'accordéon 4
        </m-accordion-stub>
      </m-accordion-group-stub>
    </p>
  </section>
</section>
`;
